<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sondos Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>731</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>DATA BASE</string>
    </property>
    <widget class="QListWidget" name="dataBase_listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>691</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>671</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Data Input</string>
    </property>
    <widget class="QPushButton" name="check_pushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="exp1_lineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string> Enter a Boolean Expression</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="exp2_lineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string> Enter a Boolean Expression</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>First Expression </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Second Expression </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>671</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Data Output</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>First Expression Validation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Second Expression Validation :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Equivalence :</string>
     </property>
    </widget>
    <widget class="QLabel" name="exp1Validation_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
    <widget class="QLabel" name="exp2Validation_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
    <widget class="QLabel" name="Equivalence_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
